src/themes/irokoto-forts/collections/table.overrides

```diff
@@ -564,3 +564,7 @@
         }
     }
 }
+
+.ui.table td {
```

Comment:
use more specific path

Reply:
I checked other table on admin side (list notice) and they have the same padding. so i add global css like this